fastavro/six.py

```diff
@@ -112,10 +112,12 @@ def py2_appendable(file_like):
             file_like.mode
         except AttributeError:
             # This is probably some io stream so we rely on its tell() working
-            if file_like.tell() != 0 and _readable(file_like):
-                return True
-            else:
-                return False
+            try:
+                if file_like.tell() != 0 and _readable(file_like):
+                    return True
+            except (OSError, IOError):
```

Comment:
In the tests you show the example of `sys.stdout.buffer` that raises `OSError`. Is there an example of something that will raise an `IOError`? Is this a python2 thing where it raised `IOError` there but on python3 it now raises `OSError`?

Reply:
Okay. Since this is the py2 section it should be safe to just catch `IOError` as the python3 case is handled in `py3_appendable`